control-plane/tests/test_phase28_endpoint_evidence_pack_validation.py

```
@@ -0,0 +1,324 @@
+        return (
+            store,
+            service,
+            promoted_case,
+            promoted_case.evidence_ids[0],
+            reviewed_at,
+        )
+
+    def test_create_endpoint_evidence_collection_request_routes_to_executor_with_bounded_scope(
+        self,
+    ) -> None:
+        _store, service, promoted_case, anchor_evidence_id, reviewed_at = (
```

Ruff (0.15.10)

[warning] Unpacked variable `reviewed_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)